src/app/views/record/record.html

```diff
@@ -70,7 +70,7 @@ <h2>{{$ctrl.record.entity}} {{$ctrl.getIdentifier($ctrl.record)}}</h2>
     </table>
   </div>
 
-  <md-divider ng-if="$ctrl.parentDetail"></md-divider>
+  <!--<md-divider ng-if="$ctrl.parentDetail"></md-divider>-->
```

Comment:
removing these dividers makes the page much harder to read imo

Reply:
Yes, i played around with this a bunch and talked to Chris about it... Since he was concerned about screen real estate i think removing them helped that issue.  I had a hard time with other methods in shrinking the tables and decided it was not worth messing with bootstrap to get the tables to condense.  In any case, i'm going to go with no dividers as i think it is just as readable.